conference.js

```diff
@@ -503,6 +503,8 @@ export default {
 
         let tryCreateLocalTracks;
 
+        const timeout = browser.isElectron() ? 15000 : 60000;
```

Comment:
Why it is different for electron? Maybe add a comment.

Reply:
Done!